src/kOS/Function/BuildList.cs

```diff
@@ -38,7 +40,7 @@ public override void Execute(SharedObjects shared)
                     list = ListValue.CreateList(shared.VolumeMgr.CurrentVolume.GetFileList());
                     break;
                 case "volumes":
-                    list = ListValue.CreateList(shared.VolumeMgr.Volumes);
+                    list = ListValue.CreateList(shared.VolumeMgr.Volumes.Values.ToList());
```

Comment:
I agree.  There's no way that could have been usable.. except that you could probably print it and thus you'd see its ToString() in the terminal.  I'm not sure what the ToString() of a dictionary is.  maybe it dumps the values out in a way that a user might have made sense of?  But anyway, yeah I agree with this change.